css-three-column-layout/styles.css

```diff
@@ -0,0 +1,56 @@
+* {
+  background-color: rgba(25, 150, 218, 255);
+  box-sizing: border-box;
+  font-family: "lato", sans-serif;
+  color: #fff;
+}
+
+.container {
+  max-width: 1000px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.column-third {
+  float: left;
+  width: 33.33%;
+  color: aliceblue;
+  box-sizing: border-box;
+  position: relative;
+}
+
+.image-container {
+  width: 128px;
+  height: 128px;
+  margin: 20px 96px 20px 96px;
+  position: relative;
+  float: left;
```

Comment:
Are these two lines necessary? Test your output and make sure. You should definitely be avoiding `float` as much as possible, so if this is needed, feel free to send me a message and we can discuss how you can accomplish your goal here differently.